apps/frontend/src/libs/helpers/cn.ts

```diff
@@ -0,0 +1,7 @@
+import clsx, { type ClassValue } from "clsx";
+
+const cn = (...classes: ClassValue[]): string => {
```

Comment:
When I looked at the name of this function, I did not get what it does. Let's name it as `getValidClassNames`